Tests/UInt128Tests/UInt128Tests.swift

```diff
@@ -646,14 +630,6 @@ class UInt128IntegerArithmeticTests: XCTestCase {
             testInteger, 7,
             "2 += 5 Doesn't Equal Expected Result"
         )
-        XCTAssertEqual(
```

Comment:
👎 

I understand why you would remove these tests, but, once again, without a major release bump I do not feel comfortable removing any functionality, so the tests need to stay. After some of your other proposed enhancements/refactors have been implemented I would be more than willing to prepare for a new major release bump to remove this functionality, as I know it is no longer an encouraged practice to have prefix and postfix incrementers/decrementers.